docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -37,6 +37,7 @@ Version Control Best Practices
 		- If developing multiple new features simultaneously or if multiple people are creating content, work should be split into separate branches. 
 		- It’s best to split into branches by independent feature units, ex: “add-QC-before-alignment”.
 		- Once your feature is stable, create a pull request to merge the branch into your main branch. Once merged, you can delete the development branch if no longer needed. 
+		- Note on GitHub repository and Docker image versioning: Many workflow repositories will contain both a Dockerfile, with instructions for building the Docker image, in addition to the workflow descriptor file (.wdl or .cwl). This adds complexity when tags for Docker images mirror tags for the GitHub repository (as is possible using quay.io, for example). On a development branch, you may want the task to refer to a development version of the Docker image (e.g., quay.io/my_account/my_image:development). This means that a perfectly-functioning development branch commit could become "incorrect" after being merged into the master branch (because the .wdl or .cwl task(s) will be referring to development Docker image version rather than an immutable version. The best current solution is to update the descriptor file just prior to (or during) the pull request so that the tasks reference the digest format of the Docker image (e.g., quay.io/my_account/my_image:f63e020c4062e0be80831a50de8640).
```

Comment:
Two nitpicks:

1. `workflow descriptor file (.wdl or .cwl)` -> should not be BD Cat language-specific. Either drop wdl and cwl, or expand to include Nextflow and Galaxy. Also suggest `file(s)` instead of `file`.
2. `development`, as in `development branch` -> I suggest `develop`, the more common GitHub branch name.

Reply:
Agreed on the first. 
Not sure about the second, I actually read this as a feature branch. 
Development might actually be better/more generic covering both